app/views/warehouse_reports/health/claims/_table.haml

```diff
@@ -1,14 +0,0 @@
-- if @reports.any?
-  %p= paginate @reports
-  %table.table.table-sm
-    %thead
-      %tr
-        %th Report
-        %th Claims before
-        %th Status
-    %tbody
-      - @reports.each do |report|
-        %tr
-          %td= link_to_if report.completed_at, 'Review Claims', warehouse_reports_health_claim_path(id: report.id)
-          %td= report.max_date
-          %td= report.status
```

Comment:
Was this deletion intentional?  I thought this was used while the report was running.

Reply:
Yeah it was intentional, it's dead code. reviewed with @awisema